src/test/scala/uk/gov/hmrc/agent/kenshoo/monitoring/MonitoringFilterSpec.scala

```diff
@@ -20,53 +20,58 @@ import akka.actor.ActorSystem
 import akka.stream.{ActorMaterializer, Materializer}
 import com.codahale.metrics.MetricRegistry
 import org.mockito.Mockito.mock
-import org.scalatest.Matchers
+import org.scalatest.matchers.must.Matchers
 import play.api.http.HttpEntity
 import play.api.mvc.{Headers, RequestHeader, ResponseHeader, Result}
 import uk.gov.hmrc.agent.kenshoo.monitoring.support.UnitSpec
 import uk.gov.hmrc.http.HeaderCarrier
+import org.scalatest.matchers.should.Matchers._
```

Comment:
Ahh, good point. Think it's because `import org.scalatest.matchers.should.Matchers._` didn't cover `MonitoringFilterTestImp with Matchers` on line 68.

Have tweaked this import